chapter_convolutional-modern/vgg_vn.md

```diff
@@ -5,37 +5,39 @@
 # Networks Using Blocks (VGG)
 -->
 
-# *dịch tiêu đề phía trên*
+# Mạng sử dụng Khối (VGG)
 :label:`sec_vgg`
 
 <!--
 While AlexNet proved that deep convolutional neural networks can achieve good results, it did not offer a general template to guide subsequent researchers in designing new networks.
 In the following sections, we will introduce several heuristic concepts commonly used to design deep networks.
 -->
 
-*dịch đoạn phía trên*
+Mặc dù AlexNet đã chứng minh rằng các mạng nơ-ron tích chập có thể đạt được kết quả tốt, nó lại không cung cấp một khuôn mẫu chung để hỗ trợ những nhà nghiên cứu về sau thiết kế các mạng mới. 
+Trong các phần tiếp theo, chúng tôi sẽ giới thiệu một số khái niệm tiến bộ được sử dụng rộng rãi trong thiết kế mạng học sâu. 
 
 <!--
 Progress in this field mirrors that in chip design where engineers went from placing transistors to logical elements to logic blocks.
 Similarly, the design of neural network architectures had grown progressively more abstract, with researchers moving from thinking in terms of
 individual neurons to whole layers, and now to blocks, repeating patterns of layers.
 -->
 
-*dịch đoạn phía trên*
+Tiến bộ trong lĩnh vực này phản chiếu cách mà các kỹ sư đi từ việc đặt các bóng bán dẫn, đến các thành phần logic và sau đó là các khối logic.  
+Tương tự như vậy, việc thiết kế các cấu trúc của mạng nơ-ron đã phát triển ngày một trừu tượng hơn, điển hình là việc các nhà nghiên cứu đã thay đổi suy nghĩ từ quy mô các nơ-ron riêng lẻ sang các tầng, và giờ đây là các khối - đại diện cho các khuôn tầng lặp lại. 
 
 <!--
 The idea of using blocks first emerged from the [Visual Geometry Group](http://www.robots.ox.ac.uk/~vgg/) (VGG) at Oxford University, in their eponymously-named VGG network.
 It is easy to implement these repeated structures in code with any modern deep learning framework by using loops and subroutines.
 -->
 
-*dịch đoạn phía trên*
-
+Ý tưởng sử dụng các khối lần đầu xuất hiện từ [Nhóm Hình Học Trực Quan](http://www.robots.ox.ac.uk/~vgg/) (VGG) tại Đại Học Oxford, trong mạng VGG được lấy từ tên của họ. 
```

Comment:
Trong ngành CS, mình hay thấy người ta dịch từ Visual và vision là thị giác (thuộc về thị giác) nhiều hơn, đặc biệt trong bối cảnh AI hiện nay. Mình gợi ý sửa lại:
```
Ý tưởng sử dụng các khối lần đầu xuất hiện trong mạng VGG được đặt cùng tên bởi [Nhóm Hình học Thị giác (VGG)](http://www.robots.ox.ac.uk/~vgg/) (VGG) thuộc Đại học Oxford.
```

Reply:
@davidnvq bạn có thể gợi ý chỉnh sửa trực quan hơn bằng [suggestion](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request) nhé.
```suggestion
Ý tưởng sử dụng các khối lần đầu xuất hiện trong mạng VGG được đặt cùng tên bởi [Nhóm Hình học Thị giác (VGG)](http://www.robots.ox.ac.uk/~vgg/) (VGG) thuộc Đại học Oxford.
```

Reply:
@davidnvq Nếu được bạn có thể sửa lại các comment với feedback suggestion như bạn @cuongvng mention ở trên không. Việc này vừa tiết kiệm thời gian, vừa tránh được confict giữa các feedbacks

Reply:
Mình thấy ko cần thiết dịch VGG lắm, nó cũng chỉ là tên của 1 cái lab thôi.